EDEngineer/Resources/Data/blueprints.json

```diff
@@ -15087,4 +15087,1324 @@
     ],
     "Grade": 5
   }
+  {
+    "Type": "Sensors",
+    "Name": "Long Range Scanner",
+    "Engineers": [ "Lori Jameson, Bill Turner, Juri Ishmaak, Lei Chung, Tiana Fortune" ],

```

Comment:
the line of engineers should look like that :
`"Engineers": [ "Zacariah Nemo", "Bill Turner" ]`

(each engineer in their own set of double quote)